internal/cli/apps.go

```diff
@@ -669,7 +667,7 @@ func openAppCmd(cli *cli) *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "Open application settings page in Auth0 Manage",
 		Long:    "Open application settings page in Auth0 Manage.",
-		Example: `auth0 apps open <id>`,
+		Example: "auth0 apps open <id>",
```

Comment:
from prev comment, If id could adopt different forms (I'm not sure) I'd suggest including some placeholders here in the example section.

Reply:
I went with `auth0 apps open <id|audience>` in https://github.com/auth0/auth0-cli/pull/253/commits/62c64d535f533809886487d159ef00c33b04881b